templates/unbound.conf.erb

```diff
@@ -1,162 +1,217 @@
 # Managed by Puppet
 #
 <%-
-  def bool2yesno (value)
-    return value ? 'yes' : 'no'
+  if @unbound_version
+    @_unbound_version = @unbound_version
+  else
+    # this happens if unbuond is not installed i.e. first run
+    @_unbound_version = '0.a'
+  end
+  def print_config(name, value, version=false )
```

Comment:
both of these functions could be converted to, well, functions.

Reply:
thanks for the review i addressed the other comments but im not sure what you meant by this one.